terra-base-jupyter/README.md

```diff
@@ -0,0 +1,91 @@
+# terra-base-jupyter image
+
+This repo contains the terra-base-jupyter image that is compatible with the
+jupyter notebook service in [Terra]("https://app.terra.bio/") called Leonardo.
+The goal is to provide a lean, GPU-enabled base image that can be use for both Terra-suported, and custom image builds.
+
+The goal is to provide the user with a base conda environment where they can install any
+package they need, while keeping the jupyter server itself in an isolated virtual environment
+that DSP devs control using poetry as the dependency manager
+
+## Image contents
+
+`terra-base-jupyter` extends an cuda 12.2.0 and Ubuntu 22.04 base image with the minimum 
+requirements necessary to set up Jupyter and provide compatibility with Leonardo and GPU machines.
```

Comment:
... curious why we are not starting with the jupyter containers? (e.g. [minimal-notebooks](https://hub.docker.com/r/jupyter/minimal-notebook) or [jupyter-docker-stacks](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html))

just curious - thanks!